src/client/libcurve_file.h

```diff
@@ -61,7 +61,8 @@ class FileClient {
      */
     virtual int Open(const std::string& filename,
                      const UserInfo_t& userinfo,
-                     const OpenFlags& openflags = {});
+                     const OpenFlags& openflags = {},
+                     const OpenFileContext_t* context = nullptr);
```

Comment:
ditto, `OpenFileContext` should not exists in user interface

Reply:
but here is not user interface.

Reply:
not exactly, I think `OpenFileContext` should hides in `FileInstance`